src/app/plugins/platform/Docker/plugins/cms/WordPress/index.ts

```diff
@@ -341,9 +390,53 @@ class WordPress extends Generator {
         { documentRoot: this.documentRoot },
       );
     }
+
+    this._writeCodeQualityConfig();
+  }
+
+  /**
+   * Write code quality configuration files for the project.
+   */
+  private _writeCodeQualityConfig(): void {
+    this.debug(
+      format.debug('Rendering .codacy.yml template to %s.'),
+      '.codacy.yml',
+    );
+    this.renderTemplate(
+      this.templatePath('_codacy.yml.ejs'),
+      this.destinationPath('.codacy.yml'),
+      {
+        documentRoot: this.documentRoot,
+        useGesso: this.usesGesso,
+      },
+    );
+
+    this.debug(
+      format.debug('Rendering phpcs.xml.dist template to %s.'),
+      'services/drupal/phpcs.xml.dist',
+    );
+    this.renderTemplate(
+      this.templatePath('phpcs.xml.dist.ejs'),
+      this.destinationPath('services/drupal/phpcs.xml.dist'),
+      {
+        documentRoot: this.documentRoot,
+      },
+    );
+
+    this.debug(
+      format.debug('Rendering .phpmd.xml.dist template to %s.'),
+      'services/drupal/.phpmd.xml.dist',
+    );
+    this.renderTemplate(
+      this.templatePath('_phpmd.xml.dist.ejs'),
+      this.destinationPath('services/drupal/.phpmd.xml.dist'),
```

Comment:
Another one that needs to be fixed for WordPress.

Reply:
Addressed in 7a1e393.